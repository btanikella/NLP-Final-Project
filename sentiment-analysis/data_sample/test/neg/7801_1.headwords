story
boring
wound
thing
happened
were
episode
people
