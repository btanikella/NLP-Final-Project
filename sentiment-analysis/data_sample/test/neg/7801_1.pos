DT CD VBZ DT JJ NN NN RB VBZ RB RB TO NN IN JJS JJ NN NNS
PRP VBZ RB JJ  IN VBG NNP NNP VBD RP IN DT NNP NNP NN VBZ NN IN NNP NNP NNP MD VB VBN IN DT JJ NN IN PRP$ NN  PRP MD VB DT NN CC DT NN CC VB IN IN DT NN NN
CD NNS IN IN CD  PRP VBD RP VBG NNS NN
JJ NN
DT RB VBN TO PRP$ NN -LRB- DT NN NN IN PRP -RRB- RB RB
EX VBD RB JJ NNS WDT CD IN PRP MD VB TO DT JJ   PRP VBD DT VBN TO VB NNP DT NNP NNP NN CC NN WDT NN VBD
DT JJ NN NN
RB DT NNS IN DT NN  IN PRP VBZ DT CD NN : WRB DT VBD  PRP VBP DT NN : CC WDT IN CC IN PRP MD VB DT JJ NNP NNP NN
