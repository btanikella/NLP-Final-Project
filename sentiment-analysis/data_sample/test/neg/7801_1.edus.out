Sentence #1 (19 tokens):
This one is a real stinker.The story just isn't up to par with most other TZ episodes.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=one CharacterOffsetBegin=9 CharacterOffsetEnd=12 PartOfSpeech=CD] [Text=is CharacterOffsetBegin=13 CharacterOffsetEnd=15 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=16 CharacterOffsetEnd=17 PartOfSpeech=DT] [Text=real CharacterOffsetBegin=18 CharacterOffsetEnd=22 PartOfSpeech=JJ] [Text=stinker.The CharacterOffsetBegin=23 CharacterOffsetEnd=34 PartOfSpeech=NN] [Text=story CharacterOffsetBegin=35 CharacterOffsetEnd=40 PartOfSpeech=NN] [Text=just CharacterOffsetBegin=41 CharacterOffsetEnd=45 PartOfSpeech=RB] [Text=is CharacterOffsetBegin=46 CharacterOffsetEnd=48 PartOfSpeech=VBZ] [Text=n't CharacterOffsetBegin=48 CharacterOffsetEnd=51 PartOfSpeech=RB] [Text=up CharacterOffsetBegin=52 CharacterOffsetEnd=54 PartOfSpeech=RB] [Text=to CharacterOffsetBegin=55 CharacterOffsetEnd=57 PartOfSpeech=TO] [Text=par CharacterOffsetBegin=58 CharacterOffsetEnd=61 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=62 CharacterOffsetEnd=66 PartOfSpeech=IN] [Text=most CharacterOffsetBegin=67 CharacterOffsetEnd=71 PartOfSpeech=JJS] [Text=other CharacterOffsetBegin=72 CharacterOffsetEnd=77 PartOfSpeech=JJ] [Text=TZ CharacterOffsetBegin=78 CharacterOffsetEnd=80 PartOfSpeech=NN] [Text=episodes CharacterOffsetBegin=81 CharacterOffsetEnd=89 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=89 CharacterOffsetEnd=90 PartOfSpeech=.] 
root(ROOT-0, story-7)
det(one-2, This-1)
nsubj(story-7, one-2)
cop(story-7, is-3)
det(story-7, a-4)
amod(story-7, real-5)
nn(story-7, stinker.The-6)
advmod(up-11, just-8)
cop(up-11, is-9)
neg(up-11, n't-10)
advmod(story-7, up-11)
prep(up-11, to-12)
pobj(to-12, par-13)
prep(up-11, with-14)
amod(episodes-18, most-15)
amod(episodes-18, other-16)
nn(episodes-18, TZ-17)
pobj(with-14, episodes-18)
punct(story-7, .-19)

Sentence #2 (49 tokens):
It's pretty boring, though seeing Peter Falk made up as a Fidel Castro lookalike is kind of amusing.Whenever Twilight Zone would be aired at an unannounced hour in my hometown, I'd grab a sandwich and a drink and settle in for some quality entertainment.
[Text=It CharacterOffsetBegin=95 CharacterOffsetEnd=97 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=97 CharacterOffsetEnd=99 PartOfSpeech=VBZ] [Text=pretty CharacterOffsetBegin=100 CharacterOffsetEnd=106 PartOfSpeech=RB] [Text=boring CharacterOffsetBegin=107 CharacterOffsetEnd=113 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=113 CharacterOffsetEnd=114 PartOfSpeech=,] [Text=though CharacterOffsetBegin=115 CharacterOffsetEnd=121 PartOfSpeech=IN] [Text=seeing CharacterOffsetBegin=122 CharacterOffsetEnd=128 PartOfSpeech=VBG] [Text=Peter CharacterOffsetBegin=129 CharacterOffsetEnd=134 PartOfSpeech=NNP] [Text=Falk CharacterOffsetBegin=135 CharacterOffsetEnd=139 PartOfSpeech=NNP] [Text=made CharacterOffsetBegin=140 CharacterOffsetEnd=144 PartOfSpeech=VBD] [Text=up CharacterOffsetBegin=145 CharacterOffsetEnd=147 PartOfSpeech=RP] [Text=as CharacterOffsetBegin=148 CharacterOffsetEnd=150 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=151 CharacterOffsetEnd=152 PartOfSpeech=DT] [Text=Fidel CharacterOffsetBegin=153 CharacterOffsetEnd=158 PartOfSpeech=NNP] [Text=Castro CharacterOffsetBegin=159 CharacterOffsetEnd=165 PartOfSpeech=NNP] [Text=lookalike CharacterOffsetBegin=166 CharacterOffsetEnd=175 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=176 CharacterOffsetEnd=178 PartOfSpeech=VBZ] [Text=kind CharacterOffsetBegin=179 CharacterOffsetEnd=183 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=184 CharacterOffsetEnd=186 PartOfSpeech=IN] [Text=amusing.Whenever CharacterOffsetBegin=187 CharacterOffsetEnd=203 PartOfSpeech=NNP] [Text=Twilight CharacterOffsetBegin=204 CharacterOffsetEnd=212 PartOfSpeech=NNP] [Text=Zone CharacterOffsetBegin=213 CharacterOffsetEnd=217 PartOfSpeech=NNP] [Text=would CharacterOffsetBegin=218 CharacterOffsetEnd=223 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=224 CharacterOffsetEnd=226 PartOfSpeech=VB] [Text=aired CharacterOffsetBegin=227 CharacterOffsetEnd=232 PartOfSpeech=VBN] [Text=at CharacterOffsetBegin=233 CharacterOffsetEnd=235 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=236 CharacterOffsetEnd=238 PartOfSpeech=DT] [Text=unannounced CharacterOffsetBegin=239 CharacterOffsetEnd=250 PartOfSpeech=JJ] [Text=hour CharacterOffsetBegin=251 CharacterOffsetEnd=255 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=256 CharacterOffsetEnd=258 PartOfSpeech=IN] [Text=my CharacterOffsetBegin=259 CharacterOffsetEnd=261 PartOfSpeech=PRP$] [Text=hometown CharacterOffsetBegin=262 CharacterOffsetEnd=270 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=270 CharacterOffsetEnd=271 PartOfSpeech=,] [Text=I CharacterOffsetBegin=272 CharacterOffsetEnd=273 PartOfSpeech=PRP] [Text='d CharacterOffsetBegin=273 CharacterOffsetEnd=275 PartOfSpeech=MD] [Text=grab CharacterOffsetBegin=276 CharacterOffsetEnd=280 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=281 CharacterOffsetEnd=282 PartOfSpeech=DT] [Text=sandwich CharacterOffsetBegin=283 CharacterOffsetEnd=291 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=292 CharacterOffsetEnd=295 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=296 CharacterOffsetEnd=297 PartOfSpeech=DT] [Text=drink CharacterOffsetBegin=298 CharacterOffsetEnd=303 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=304 CharacterOffsetEnd=307 PartOfSpeech=CC] [Text=settle CharacterOffsetBegin=308 CharacterOffsetEnd=314 PartOfSpeech=VB] [Text=in CharacterOffsetBegin=315 CharacterOffsetEnd=317 PartOfSpeech=IN] [Text=for CharacterOffsetBegin=318 CharacterOffsetEnd=321 PartOfSpeech=IN] [Text=some CharacterOffsetBegin=322 CharacterOffsetEnd=326 PartOfSpeech=DT] [Text=quality CharacterOffsetBegin=327 CharacterOffsetEnd=334 PartOfSpeech=NN] [Text=entertainment CharacterOffsetBegin=335 CharacterOffsetEnd=348 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=348 CharacterOffsetEnd=349 PartOfSpeech=.] 
root(ROOT-0, boring-4)
nsubj(boring-4, It-1)
cop(boring-4, 's-2)
advmod(boring-4, pretty-3)
punct(boring-4, ,-5)
prep(boring-4, though-6)
pcomp(though-6, seeing-7)
nn(Falk-9, Peter-8)
nsubj(made-10, Falk-9)
ccomp(seeing-7, made-10)
prt(made-10, up-11)
mark(kind-18, as-12)
det(lookalike-16, a-13)
nn(lookalike-16, Fidel-14)
nn(lookalike-16, Castro-15)
nsubj(kind-18, lookalike-16)
cop(kind-18, is-17)
nsubjpass(aired-25, kind-18)
prep(kind-18, of-19)
nn(Zone-22, amusing.Whenever-20)
nn(Zone-22, Twilight-21)
pobj(of-19, Zone-22)
aux(aired-25, would-23)
auxpass(aired-25, be-24)
ccomp(made-10, aired-25)
prep(aired-25, at-26)
det(hour-29, an-27)
amod(hour-29, unannounced-28)
pobj(at-26, hour-29)
prep(aired-25, in-30)
poss(hometown-32, my-31)
pobj(in-30, hometown-32)
punct(aired-25, ,-33)
nsubj(grab-36, I-34)
aux(grab-36, 'd-35)
ccomp(aired-25, grab-36)
det(sandwich-38, a-37)
dobj(grab-36, sandwich-38)
cc(grab-36, and-39)
det(drink-41, a-40)
conj(grab-36, drink-41)
cc(drink-41, and-42)
conj(drink-41, settle-43)
prep(drink-41, in-44)
pcomp(in-44, for-45)
det(entertainment-48, some-46)
nn(entertainment-48, quality-47)
pobj(for-45, entertainment-48)
punct(boring-4, .-49)

Sentence #3 (13 tokens):
9 times out of 10, it wound up being THIS episode!
[Text=9 CharacterOffsetBegin=354 CharacterOffsetEnd=355 PartOfSpeech=CD] [Text=times CharacterOffsetBegin=356 CharacterOffsetEnd=361 PartOfSpeech=NNS] [Text=out CharacterOffsetBegin=362 CharacterOffsetEnd=365 PartOfSpeech=IN] [Text=of CharacterOffsetBegin=366 CharacterOffsetEnd=368 PartOfSpeech=IN] [Text=10 CharacterOffsetBegin=369 CharacterOffsetEnd=371 PartOfSpeech=CD] [Text=, CharacterOffsetBegin=371 CharacterOffsetEnd=372 PartOfSpeech=,] [Text=it CharacterOffsetBegin=373 CharacterOffsetEnd=375 PartOfSpeech=PRP] [Text=wound CharacterOffsetBegin=376 CharacterOffsetEnd=381 PartOfSpeech=VBD] [Text=up CharacterOffsetBegin=382 CharacterOffsetEnd=384 PartOfSpeech=RP] [Text=being CharacterOffsetBegin=385 CharacterOffsetEnd=390 PartOfSpeech=VBG] [Text=THIS CharacterOffsetBegin=391 CharacterOffsetEnd=395 PartOfSpeech=NNS] [Text=episode CharacterOffsetBegin=396 CharacterOffsetEnd=403 PartOfSpeech=NN] [Text=! CharacterOffsetBegin=403 CharacterOffsetEnd=404 PartOfSpeech=.] 
root(ROOT-0, wound-8)
number(times-2, 9-1)
dep(wound-8, times-2)
prep(times-2, out-3)
pcomp(out-3, of-4)
pobj(of-4, 10-5)
punct(wound-8, ,-6)
nsubj(wound-8, it-7)
prt(wound-8, up-9)
cop(episode-12, being-10)
nn(episode-12, THIS-11)
xcomp(wound-8, episode-12)
punct(wound-8, !-13)

Sentence #4 (3 tokens):
Strange thing?
[Text=Strange CharacterOffsetBegin=409 CharacterOffsetEnd=416 PartOfSpeech=JJ] [Text=thing CharacterOffsetBegin=417 CharacterOffsetEnd=422 PartOfSpeech=NN] [Text=? CharacterOffsetBegin=422 CharacterOffsetEnd=423 PartOfSpeech=.] 
root(ROOT-0, thing-2)
amod(thing-2, Strange-1)
punct(thing-2, ?-3)

Sentence #5 (16 tokens):
This often happened to my mom (another TZ fan like me) as well.
[Text=This CharacterOffsetBegin=428 CharacterOffsetEnd=432 PartOfSpeech=DT] [Text=often CharacterOffsetBegin=433 CharacterOffsetEnd=438 PartOfSpeech=RB] [Text=happened CharacterOffsetBegin=439 CharacterOffsetEnd=447 PartOfSpeech=VBN] [Text=to CharacterOffsetBegin=448 CharacterOffsetEnd=450 PartOfSpeech=TO] [Text=my CharacterOffsetBegin=451 CharacterOffsetEnd=453 PartOfSpeech=PRP$] [Text=mom CharacterOffsetBegin=454 CharacterOffsetEnd=457 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=458 CharacterOffsetEnd=459 PartOfSpeech=-LRB-] [Text=another CharacterOffsetBegin=459 CharacterOffsetEnd=466 PartOfSpeech=DT] [Text=TZ CharacterOffsetBegin=467 CharacterOffsetEnd=469 PartOfSpeech=NN] [Text=fan CharacterOffsetBegin=470 CharacterOffsetEnd=473 PartOfSpeech=NN] [Text=like CharacterOffsetBegin=474 CharacterOffsetEnd=478 PartOfSpeech=IN] [Text=me CharacterOffsetBegin=479 CharacterOffsetEnd=481 PartOfSpeech=PRP] [Text=-RRB- CharacterOffsetBegin=481 CharacterOffsetEnd=482 PartOfSpeech=-RRB-] [Text=as CharacterOffsetBegin=483 CharacterOffsetEnd=485 PartOfSpeech=RB] [Text=well CharacterOffsetBegin=486 CharacterOffsetEnd=490 PartOfSpeech=RB] [Text=. CharacterOffsetBegin=490 CharacterOffsetEnd=491 PartOfSpeech=.] 
root(ROOT-0, happened-3)
nsubj(happened-3, This-1)
advmod(happened-3, often-2)
prep(happened-3, to-4)
poss(mom-6, my-5)
pobj(to-4, mom-6)
punct(fan-10, -LRB--7)
det(fan-10, another-8)
amod(fan-10, TZ-9)
appos(mom-6, fan-10)
prep(fan-10, like-11)
pobj(like-11, me-12)
punct(fan-10, -RRB--13)
advmod(well-15, as-14)
advmod(mom-6, well-15)
punct(happened-3, .-16)

Sentence #6 (34 tokens):
There were so many times that one of us would say to the other, "I was all set to watch _The Twilight Zone_ and guess which episode was on?
[Text=There CharacterOffsetBegin=496 CharacterOffsetEnd=501 PartOfSpeech=EX] [Text=were CharacterOffsetBegin=502 CharacterOffsetEnd=506 PartOfSpeech=VBD] [Text=so CharacterOffsetBegin=507 CharacterOffsetEnd=509 PartOfSpeech=RB] [Text=many CharacterOffsetBegin=510 CharacterOffsetEnd=514 PartOfSpeech=JJ] [Text=times CharacterOffsetBegin=515 CharacterOffsetEnd=520 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=521 CharacterOffsetEnd=525 PartOfSpeech=WDT] [Text=one CharacterOffsetBegin=526 CharacterOffsetEnd=529 PartOfSpeech=CD] [Text=of CharacterOffsetBegin=530 CharacterOffsetEnd=532 PartOfSpeech=IN] [Text=us CharacterOffsetBegin=533 CharacterOffsetEnd=535 PartOfSpeech=PRP] [Text=would CharacterOffsetBegin=536 CharacterOffsetEnd=541 PartOfSpeech=MD] [Text=say CharacterOffsetBegin=542 CharacterOffsetEnd=545 PartOfSpeech=VB] [Text=to CharacterOffsetBegin=546 CharacterOffsetEnd=548 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=549 CharacterOffsetEnd=552 PartOfSpeech=DT] [Text=other CharacterOffsetBegin=553 CharacterOffsetEnd=558 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=558 CharacterOffsetEnd=559 PartOfSpeech=,] [Text=`` CharacterOffsetBegin=560 CharacterOffsetEnd=561 PartOfSpeech=``] [Text=I CharacterOffsetBegin=561 CharacterOffsetEnd=562 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=563 CharacterOffsetEnd=566 PartOfSpeech=VBD] [Text=all CharacterOffsetBegin=567 CharacterOffsetEnd=570 PartOfSpeech=DT] [Text=set CharacterOffsetBegin=571 CharacterOffsetEnd=574 PartOfSpeech=VBN] [Text=to CharacterOffsetBegin=575 CharacterOffsetEnd=577 PartOfSpeech=TO] [Text=watch CharacterOffsetBegin=578 CharacterOffsetEnd=583 PartOfSpeech=VB] [Text=_ CharacterOffsetBegin=584 CharacterOffsetEnd=585 PartOfSpeech=NNP] [Text=The CharacterOffsetBegin=585 CharacterOffsetEnd=588 PartOfSpeech=DT] [Text=Twilight CharacterOffsetBegin=589 CharacterOffsetEnd=597 PartOfSpeech=NNP] [Text=Zone CharacterOffsetBegin=598 CharacterOffsetEnd=602 PartOfSpeech=NNP] [Text=_ CharacterOffsetBegin=602 CharacterOffsetEnd=603 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=604 CharacterOffsetEnd=607 PartOfSpeech=CC] [Text=guess CharacterOffsetBegin=608 CharacterOffsetEnd=613 PartOfSpeech=NN] [Text=which CharacterOffsetBegin=614 CharacterOffsetEnd=619 PartOfSpeech=WDT] [Text=episode CharacterOffsetBegin=620 CharacterOffsetEnd=627 PartOfSpeech=NN] [Text=was CharacterOffsetBegin=628 CharacterOffsetEnd=631 PartOfSpeech=VBD] [Text=on CharacterOffsetBegin=632 CharacterOffsetEnd=634 PartOfSpeech=IN] [Text=? CharacterOffsetBegin=634 CharacterOffsetEnd=635 PartOfSpeech=.] 
root(ROOT-0, were-2)
expl(were-2, There-1)
advmod(times-5, so-3)
amod(times-5, many-4)
nsubj(were-2, times-5)
dobj(say-11, that-6)
nsubj(say-11, one-7)
prep(one-7, of-8)
pobj(of-8, us-9)
aux(say-11, would-10)
rcmod(times-5, say-11)
prep(say-11, to-12)
det(other-14, the-13)
pobj(to-12, other-14)
punct(were-2, ,-15)
punct(were-2, ``-16)
nsubjpass(set-20, I-17)
auxpass(set-20, was-18)
dep(set-20, all-19)
ccomp(were-2, set-20)
aux(watch-22, to-21)
xcomp(set-20, watch-22)
dobj(watch-22, _-23)
det(_-27, The-24)
nn(_-27, Twilight-25)
nn(_-27, Zone-26)
dep(_-23, _-27)
cc(_-27, and-28)
conj(_-27, guess-29)
det(episode-31, which-30)
nsubj(was-32, episode-31)
rcmod(_-27, was-32)
advmod(was-32, on-33)
punct(were-2, ?-34)

Sentence #7 (6 tokens):
The Cuban Dictator episode!"
[Text=The CharacterOffsetBegin=640 CharacterOffsetEnd=643 PartOfSpeech=DT] [Text=Cuban CharacterOffsetBegin=644 CharacterOffsetEnd=649 PartOfSpeech=JJ] [Text=Dictator CharacterOffsetBegin=650 CharacterOffsetEnd=658 PartOfSpeech=NN] [Text=episode CharacterOffsetBegin=659 CharacterOffsetEnd=666 PartOfSpeech=NN] [Text=! CharacterOffsetBegin=666 CharacterOffsetEnd=667 PartOfSpeech=.] [Text='' CharacterOffsetBegin=667 CharacterOffsetEnd=668 PartOfSpeech=''] 
root(ROOT-0, episode-4)
det(episode-4, The-1)
amod(episode-4, Cuban-2)
nn(episode-4, Dictator-3)
punct(episode-4, !-5)
punct(episode-4, ''-6)

Sentence #8 (37 tokens):
Obviously some people like this episode, as it gets a 6.7 rating ... how that happened, I have no idea - but THAT in and of itself would make a great Twilight Zone plot!
[Text=Obviously CharacterOffsetBegin=673 CharacterOffsetEnd=682 PartOfSpeech=RB] [Text=some CharacterOffsetBegin=683 CharacterOffsetEnd=687 PartOfSpeech=DT] [Text=people CharacterOffsetBegin=688 CharacterOffsetEnd=694 PartOfSpeech=NNS] [Text=like CharacterOffsetBegin=695 CharacterOffsetEnd=699 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=700 CharacterOffsetEnd=704 PartOfSpeech=DT] [Text=episode CharacterOffsetBegin=705 CharacterOffsetEnd=712 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=712 CharacterOffsetEnd=713 PartOfSpeech=,] [Text=as CharacterOffsetBegin=714 CharacterOffsetEnd=716 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=717 CharacterOffsetEnd=719 PartOfSpeech=PRP] [Text=gets CharacterOffsetBegin=720 CharacterOffsetEnd=724 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=725 CharacterOffsetEnd=726 PartOfSpeech=DT] [Text=6.7 CharacterOffsetBegin=727 CharacterOffsetEnd=730 PartOfSpeech=CD] [Text=rating CharacterOffsetBegin=731 CharacterOffsetEnd=737 PartOfSpeech=NN] [Text=... CharacterOffsetBegin=738 CharacterOffsetEnd=741 PartOfSpeech=:] [Text=how CharacterOffsetBegin=742 CharacterOffsetEnd=745 PartOfSpeech=WRB] [Text=that CharacterOffsetBegin=746 CharacterOffsetEnd=750 PartOfSpeech=DT] [Text=happened CharacterOffsetBegin=751 CharacterOffsetEnd=759 PartOfSpeech=VBD] [Text=, CharacterOffsetBegin=759 CharacterOffsetEnd=760 PartOfSpeech=,] [Text=I CharacterOffsetBegin=761 CharacterOffsetEnd=762 PartOfSpeech=PRP] [Text=have CharacterOffsetBegin=763 CharacterOffsetEnd=767 PartOfSpeech=VBP] [Text=no CharacterOffsetBegin=768 CharacterOffsetEnd=770 PartOfSpeech=DT] [Text=idea CharacterOffsetBegin=771 CharacterOffsetEnd=775 PartOfSpeech=NN] [Text=- CharacterOffsetBegin=776 CharacterOffsetEnd=777 PartOfSpeech=:] [Text=but CharacterOffsetBegin=778 CharacterOffsetEnd=781 PartOfSpeech=CC] [Text=THAT CharacterOffsetBegin=782 CharacterOffsetEnd=786 PartOfSpeech=WDT] [Text=in CharacterOffsetBegin=787 CharacterOffsetEnd=789 PartOfSpeech=IN] [Text=and CharacterOffsetBegin=790 CharacterOffsetEnd=793 PartOfSpeech=CC] [Text=of CharacterOffsetBegin=794 CharacterOffsetEnd=796 PartOfSpeech=IN] [Text=itself CharacterOffsetBegin=797 CharacterOffsetEnd=803 PartOfSpeech=PRP] [Text=would CharacterOffsetBegin=804 CharacterOffsetEnd=809 PartOfSpeech=MD] [Text=make CharacterOffsetBegin=810 CharacterOffsetEnd=814 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=815 CharacterOffsetEnd=816 PartOfSpeech=DT] [Text=great CharacterOffsetBegin=817 CharacterOffsetEnd=822 PartOfSpeech=JJ] [Text=Twilight CharacterOffsetBegin=823 CharacterOffsetEnd=831 PartOfSpeech=NNP] [Text=Zone CharacterOffsetBegin=832 CharacterOffsetEnd=836 PartOfSpeech=NNP] [Text=plot CharacterOffsetBegin=837 CharacterOffsetEnd=841 PartOfSpeech=NN] [Text=! CharacterOffsetBegin=841 CharacterOffsetEnd=842 PartOfSpeech=.] 
root(ROOT-0, people-3)
advmod(people-3, Obviously-1)
det(people-3, some-2)
prep(people-3, like-4)
det(episode-6, this-5)
pobj(like-4, episode-6)
punct(people-3, ,-7)
mark(gets-10, as-8)
nsubj(gets-10, it-9)
dep(people-3, gets-10)
det(rating-13, a-11)
num(rating-13, 6.7-12)
dobj(gets-10, rating-13)
punct(people-3, ...-14)
advmod(happened-17, how-15)
nsubj(happened-17, that-16)
advcl(have-20, happened-17)
punct(have-20, ,-18)
nsubj(have-20, I-19)
dep(people-3, have-20)
det(idea-22, no-21)
dobj(have-20, idea-22)
punct(idea-22, --23)
cc(idea-22, but-24)
nsubj(make-31, THAT-25)
prep(THAT-25, in-26)
cc(in-26, and-27)
conj(in-26, of-28)
pobj(of-28, itself-29)
aux(make-31, would-30)
conj(idea-22, make-31)
det(plot-36, a-32)
amod(plot-36, great-33)
nn(plot-36, Twilight-34)
nn(plot-36, Zone-35)
dobj(make-31, plot-36)
punct(people-3, !-37)

